src/components/PlayerConfig.js

```diff
@@ -41,93 +41,94 @@ export const playerConfig = {
-    organisation: ["Test Channel"],
-    showNotification: true,
-    language: ["English"],
-    mimeType: "application/pdf",
-    variants: {
-      full: {
-        ecarUrl:
-          "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/do_2141610327664312321258/pdf-content_1728997671512_do_2141610327664312321258_1.ecar",
-        size: "5483",
-      },
-      spine: {
-        ecarUrl:
-          "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/do_2141610327664312321258/pdf-content_1728997671759_do_2141610327664312321258_1_SPINE.ecar",
-        size: "4408",
-      },
-    },
-    objectType: "Content",
-    appIcon:
-      "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/do_2141610327664312321258/artifact/do_2141632198427934721299_1728908672301_sunbird.thumb.jpeg",
-    primaryCategory: "Learning Resource",
-    contentEncoding: "identity",
-    artifactUrl:
-      "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/assets/do_214168086108602368114/3-mb-sample-pdf-file.pdf",
-    contentType: "Resource",
-    trackable: {
-      enabled: "No",
-      autoBatch: "No",
-    },
-    identifier: "do_2141610327664312321258",
-    lastUpdatedBy: "5afb0c71-5e85-46f6-8780-3059cbb7bbf9",
-    audience: ["Student"],
-    visibility: "Default",
-    discussionForum: {
-      enabled: "No",
-    },
-    mediaType: "content",
-    osId: "org.ekstep.quiz.app",
-    languageCode: ["en"],
-    lastPublishedBy: "4cd4c690-eab6-4938-855a-447c7b1b8ea9",
-    version: 2,
-    pragma: ["external"],
-    license: "CC BY 4.0",
-    prevState: "Draft",
-    size: 3028.0,
-    lastPublishedOn: "2024-10-15T13:07:50.610+0000",
-    name: "PDF Content",
-    status: "Live",
-    code: "27a5ae06-ca31-443e-8a5b-59235ece065e",
-    interceptionPoints: {},
-    credentials: {
-      enabled: "No",
-    },
-    prevStatus: "Processing",
-    posterImage:
-      "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/do_2141632198427934721299/artifact/do_2141632198427934721299_1728908672301_sunbird.jpeg",
-    idealScreenSize: "normal",
-    createdOn: "2024-10-11T10:14:55.136+0000",
-    contentDisposition: "inline",
-    lastUpdatedOn: "2024-10-15T13:07:52.009+0000",
-    dialcodeRequired: "No",
-    lastStatusChangedOn: "2024-10-15T13:07:52.009+0000",
-    createdFor: ["test-k12-channel"],
-    creator: "Rahul Tekdi",
-    os: ["All"],
-    se_FWIds: ["test_k12_framework"],
-    pkgVersion: 1,
-    versionKey: "1728908675009",
-    idealScreenDensity: "hdpi",
-    framework: "test_k12_framework",
-    createdBy: "5afb0c71-5e85-46f6-8780-3059cbb7bbf9",
-    compatibilityLevel: 4,
-    resourceType: "Learn",
+    // ownershipType: ["createdBy"],
+    // copyright: "Test Channel",
+    // previewUrl:
+    //   "https://knowlg-public.s3-ap-south-1.amazonaws.com/content/assets/do_2141610327664312321258/sample.pdf",
+    // channel: "test-k12-channel",
+    // downloadUrl:
```

Comment:
@suvarnakale remove this metadata and keep separate metadata for both pdf and video type